### ISSUE TITLE
[Tests] System.Net.Security.Tests.LoggingTest.EventSource_EventsRaisedAsExpected Unhandled exception Unstable on Windows 11

### ISSUE THREAD

**Issue by mdh1418** [dotnet/runtime#65279]
```
Unhandled exception. Microsoft.DotNet.XUnitExtensions.SkipTestException: Unstable on Windows 11
   at System.Net.Security.Tests.SslStreamStreamToStreamTest.SslStream_StreamToStream_Authentication_Success(X509Certificate serverCert, X509Certificate clientCert) in /_/src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs:line 79
   at System.Net.Security.Tests.LoggingTest.<>c.<EventSource_EventsRaisedAsExpected>b__1_1() in /_/src/libraries/System.Net.Security/tests/FunctionalTests/LoggingTest.cs:line 45
   at System.Diagnostics.Tracing.TestEventListener.RunWithCallback(Action`1 handler, Action body) in /_/src/libraries/Common/tests/System/Diagnostics/Tracing/TestEventListener.cs:line 111
   at System.Net.Security.Tests.LoggingTest.<>c.<EventSource_EventsRaisedAsExpected>b__1_0() in /_/src/libraries/System.Net.Security/tests/FunctionalTests/LoggingTest.cs:line 41
--- End of stack trace from previous location ---
   at Microsoft.DotNet.RemoteExecutor.Program.Main(String[] args) in /_/src/Microsoft.DotNet.RemoteExecutor/src/Program.cs:line 94
    System.Net.Security.Tests.LoggingTest.EventSource_EventsRaisedAsExpected [FAIL]
      Microsoft.DotNet.RemoteExecutor.RemoteExecutionException : Remote process failed with an unhandled exception.
      Stack Trace:
        
        Child exception:
          Microsoft.DotNet.XUnitExtensions.SkipTestException: Unstable on Windows 11
        /_/src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs(79,0): at System.Net.Security.Tests.SslStreamStreamToStreamTest.SslStream_StreamToStream_Authentication_Success(X509Certificate serverCert, X509Certificate clientCert)
        /_/src/libraries/System.Net.Security/tests/FunctionalTests/LoggingTest.cs(45,0): at System.Net.Security.Tests.LoggingTest.<>c.<EventSource_EventsRaisedAsExpected>b__1_1()
        /_/src/libraries/Common/tests/System/Diagnostics/Tracing/TestEventListener.cs(111,0): at System.Diagnostics.Tracing.TestEventListener.RunWithCallback(Action`1 handler, Action body)
        /_/src/libraries/System.Net.Security/tests/FunctionalTests/LoggingTest.cs(41,0): at System.Net.Security.Tests.LoggingTest.<>c.<EventSource_EventsRaisedAsExpected>b__1_0()
        
        Child process:
          System.Net.Security.Tests, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51 System.Net.Security.Tests.LoggingTest+<>c Void <EventSource_EventsRaisedAsExpected>b__1_0()
```
Build: https://dev.azure.com/dnceng/public/_build/results?buildId=1606293&view=logs&j=457f7e88-dfa2-5bd9-f871-fdf124c2477d&t=bfe52dfb-2099-5c7f-ee52-70a1d81c544e&s=96ac2280-8cb4-5df5-99de-dd2da759617d
Log: https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-64952-merge-bb663c35c087459bb8/System.Net.Security.Tests/3/console.4cc83c6b.log?sv=2019-07-07&se=2022-03-03T01%3A58%3A49Z&sr=c&sp=rl&sig=8twS3bwRjJKf0aJthzMSrl8XnyO1gthfCqRNFarRKm0%3D

**Comment by wfurt** [comment]
We can certainly add notes to tests used indirectly to make sure the relation is clear when updating. 
Alternatively we can re-factor some of the code so the guts can be used by the logging tests directly instead of just  calling another test.

**Comment by akoeplinger** [comment]
@MihaZupan @wfurt I'm seeing this failing on release/6.0 the same way, can we backport this fix?

**Comment by rzikm** [comment]
> @MihaZupan Miha Zupan FTE @wfurt Tomas Weinfurt FTE I'm seeing this failing on release/6.0 the same way, can we backport the fix?

I have a WIP PR which backports bunch of test fixes to 6.0, should be up soon
